### ISSUE TITLE
Cannot run the server in dev

### ISSUE THREAD

**Issue by hafizio** [glinton/magento-test#1]
Steps to reproduce
--
1. I cloned the `magento-test` repo.
2. Follow the installation steps.
3. `nanobox run`
4. `start-apache`

Expected result
--
1. The application is accessible at a local port

Actual result
--
1. Error message appears: "Internal Server Error. The server encountered an internal error or misconfiguration and was unable to complete your request.".

2. Log
```
[Thu Apr 27 12:25:14 2017] [notice] FastCGI: process manager initialized (pid 173)
[Thu Apr 27 12:25:14 2017] [notice] Apache/2.2.32 (Unix) mod_fastcgi/2.4.6 configured -- resuming normal operations
[Thu Apr 27 12:25:59 2017] [error] [client 172.19.255.254] (2)No such file or directory: FastCGI: failed to connect to server "/data/libexec/cgi-bin/php-cgi": connect() failed
[Thu Apr 27 12:25:59 2017] [error] [client 172.19.255.254] FastCGI: incomplete headers (0 bytes) received from server "/data/libexec/cgi-bin/php-cgi"
[Thu Apr 27 12:26:00 2017] [error] [client 172.19.255.254] (2)No such file or directory: FastCGI: failed to connect to server "/data/libexec/cgi-bin/php-cgi": connect() failed, referer: http://172.19.0.14/
[Thu Apr 27 12:26:00 2017] [error] [client 172.19.255.254] FastCGI: incomplete headers (0 bytes) received from server "/data/libexec/cgi-bin/php-cgi", referer: http://172.19.0.14/
```

From the docs in the php engine, the FastCGI module is included by default. Am I missing anything here?

Thanks!

**Comment by glinton** [comment]
Woah, sorry for the delay, I'm not sure how I missed this.. 

FastCGI is included by default, but it isn't started. I believe if you do a `nanobox run` in another terminal, then run `start-php`, magento will run for you in dev.  Or just use `php-server` to start both apache and php for your step `4`.

**Comment by glinton** [comment]
This should be prevented and resolved with 386587cb5fa069c63b206e33b93093a84a8e90bd. Re-open if you are still having issues.
